packages/core-data/src/resolvers.js

```diff
@@ -993,20 +993,25 @@ export const getRevision =
 export const getRegisteredPostMeta =
 	( postType ) =>
 	async ( { dispatch, resolveSelect } ) => {
+		let options;
 		try {
 			const {
 				rest_namespace: restNamespace = 'wp/v2',
 				rest_base: restBase,
 			} = ( await resolveSelect.getPostType( postType ) ) || {};
-			const options = await apiFetch( {
+			options = await apiFetch( {
 				path: `${ restNamespace }/${ restBase }/?context=edit`,
 				method: 'OPTIONS',
 			} );
+		} catch ( error ) {
+			// Do nothing if the request comes back with an API error.
+			return;
+		}
+
+		if ( options ) {
```

Comment:
I followed the same pattern as other resolvers like [this one](https://github.com/WordPress/gutenberg/blob/3fa2af18a0ff8d8669f49fb309b3fd9e7864a0ae/packages/core-data/src/resolvers.js#L480-L491), [this one](https://github.com/WordPress/gutenberg/blob/3fa2af18a0ff8d8669f49fb309b3fd9e7864a0ae/packages/core-data/src/resolvers.js#L574-L585), or [this one](https://github.com/WordPress/gutenberg/blob/3fa2af18a0ff8d8669f49fb309b3fd9e7864a0ae/packages/core-data/src/resolvers.js#L855-L864).

I hope that's fine.